docs/source/release/v6.12.0/Inelastic/Algorithms/New_features/38196_CreateMonteCarloWorkspace.rst

```diff
@@ -0,0 +1 @@
+- New algorithm that creates a randomly simulated workspace by sampling from the probability distribution of input data.
```

Comment:
This needs to name the new algorithm and link to it (have a look at some other release notes for examples)